src/math/trigonometry.js

```diff
@@ -303,13 +298,12 @@ p5.prototype.radians = angle => angle * constants.DEG_TO_RAD;
  *   angleMode(RADIANS); // Change the mode to RADIANS
  *   rotate(a); // variable a stays the same
  *   rect(-40, -5, 20, 10); // Smaller rectangle is rotating in radians
+ *   describe(`40×10 rect in center rotates with mouse moves.
```

Comment:
@Zearin could you make a new PR correcting this to 40 by 10. Screen readers would otherwise read this as 40 ex 10